### PR TITLE
OTC-263: fix timeout issue on GetControlNumber

### PR DESCRIPTION
**Pull request by sniedzielski**
related to TICKET: https://openimis.atlassian.net/browse/OTC-263

Changes:
1. change certPass hardcoded - current one caused issues because that one was realized to be wrong during testing.
2. include also some changes from ticket https://openimis.atlassian.net/browse/OTC-259 - move hardcoded path to appsettings.json - the only difference is using in that PR "Path.Combine". 
3. Removed generating control number Randomly in ImisBasePayment.cs "PostReqControlNumber".

### CODE REVIEW

File: ImisRestApi/Escape/Payment/Data/GepgUtility.cs [hunk 1]
```diff
@@ -26,7 +26,7 @@ public class GepgUtility
         string GepgPublicCertStorePath = string.Empty;
         string GepgPayCertStorePath = string.Empty;
 
-        string CertPass = "HPSS1234";
+        string CertPass = "passpass";//HPSS1234
```
Comment (delcroip) [inline ImisRestApi/Escape/Payment/Data/GepgUtility.cs:29]: We need to add them as config

**Review by delcroip — CHANGES_REQUESTED** [review]

File: ImisRestApi/Escape/Payment/Data/GepgUtility.cs (hunk 1 above)
Comment (sniedzielski) [reply ImisRestApi/Escape/Payment/Data/GepgUtility.cs:29]: Ok.

Comment (sniedzielski) [reply ImisRestApi/Escape/Payment/Data/GepgUtility.cs:29]: Then the same thing with "GepgCertPass"

Comment (sniedzielski) [reply ImisRestApi/Escape/Payment/Data/GepgUtility.cs:29]: I will add them to a config

Comment (sniedzielski) [reply ImisRestApi/Escape/Payment/Data/GepgUtility.cs]: @delcroip changed, you can recheck this.

**Comment by sniedzielski** [comment]
@delcroip @dragos-dobre  As we discussed today I removed this randomly generated controlNumber in that commit https://github.com/openimis/rest_api_c-sharp/pull/58/commits/4cb56dd0fcc3ddbdd91f1f6ea3a2178bcc5e8c6e
1_have_base_controller
when the CHF "PostReqControlNumber" is used then in return we go into "ImisBasePayment.cs" "PostReqControlNumber" where we have this fake generator. 
1_b_have_base_controller

In SaveIntent in "PaymentLogic.cs" code checks if ControlNumberExists etc
1_c_have_base_controller

File: ImisRestApi/appsettings.json [hunk 2]
```diff
@@ -20,7 +20,13 @@
...
       "SubSpCode": 1000,
       "Url": "http://154.118.230.18:80",
-      "GfsCode": [ "300106", "300107", "300109" ]
+      "GfsCode": [ "300106", "300107", "300109" ],
+      "PrivateStorePath": "\\Escape\\Payment\\Certificates\\gepgclientprivatekey.pfx",
+      "PublicStorePath": "\\Escape\\Payment\\Certificates\\gepgclientpubliccertificate.pfx",
+      "GepgPublicCertStorePath": "\\Escape\\Payment\\Certificates\\gepgpubliccertificatetoclients.pfx",
+      "GepgPayCertStorePath": "\\Escape\\Payment\\Certificates\\gepgpaypubliccertificate.pfx",
+      "CertPass": "passpass",
+      "GepgCertPass": "gepg@2018"
```
Comment (dragos-dobre) [inline ImisRestApi/appsettings.json:29]: change this to a generic pass. this is same as original.

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

**Pull request merged**